tests/api/test_discount.py

```diff
@@ -731,12 +732,22 @@ def test_sale_remove_no_catalogues(
 @pytest.mark.parametrize(
     "voucher_filter, start_date, end_date, count",
     [
-        ({"status": "ACTIVE"}, date(2015, 1, 1), date(2020, 1, 1), 2),
-        ({"status": "EXPIRED"}, date(2015, 1, 1), date(2018, 1, 1), 1),
+        (
+            {"status": "ACTIVE"},
+            timezone.now().replace(year=2015, month=1, day=1),
+            timezone.now().replace(year=2020, month=1, day=1),
```

Comment:
And because you are here... I assume it is a good time to fix this test. It will fail after 1-Jan-2020.  Let's change the datetime to a relative.